app/views/courses/entry_requirements/_subject_choices.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="govuk-form-group govuk-!-margin-bottom-8" data-qa="course__<%= subject %>_requirements">
+  <fieldset class="govuk-fieldset">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+      <h2 class="govuk-fieldset__heading">
+        <%= subject.to_s.titleize %> GCSE
+      </h2>
+    </legend>
+    <div class="govuk-radios govuk-radios--conditional">
+      <% [
+          [
+            '1. Must have (least flexible)',
+            'must_have_qualification_at_application_time',
+            'UCAS will block applications from candidates who haven’t gained their GCSE yet or who need to take an equivalency test.'
+          ],
+          [
+            '2: Taking',
+            'expect_to_achieve_before_training_begins',
+            'You will consider candidates with a pending GCSE. But UCAS will block applications from someone who needs to take an equivalency test.'
+          ],
+          [
+            '3: Equivalence test',
+            'equivalence_test',
+            'You will consider candidates who need to take an equivalency test as well as those with a pending GCSE.'
+          ],
+          [
+            'No GCSE requirement',
+            'not_required'
```

Comment:
We might want to use `not_set` here instead (ie nil instead of 9):
https://bat-design-history.herokuapp.com/publish-teacher-training/specific-requirements#UCAS-code-explanations